f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py

```diff
@@ -397,6 +400,14 @@ def __init__(self, conf, registerOpts=True):
                  % (len(self.__bigips), self.conf.icontrol_username))
         LOG.info('iControlDriver dynamic agent configurations:%s'
                  % self.agent_configurations)
+
+        # read enhance services definitions
+        self.esd = EsdTagProcessor('/etc/neutron/services/f5/esd/')
```

Comment:
Hard coding this path might make it more difficult to move the configuartion.  I think we have a oslo_config variable that defines the location of the ini file.  So let's follow the same convention.